README.md

```diff
@@ -131,3 +131,16 @@ If you are using Atom make sure you have installed these packages:
 * [linter](https://atom.io/packages/linter)
 * [linter-eslint](https://atom.io/packages/linter-eslint)
 * [linter-less](https://atom.io/packages/linter-less)
+
+
+# Firefox development
+
+Install [web-ext](https://github.com/mozilla/web-ext)
+
+    $ npm i -g web-ext
+
+Start watcher:
+
+    $ cd app && web-ext run
```

Comment:
@kelonye is there something similar for chrome?

Reply:
they're working on adding chrome support but i'll investigate if there're existing similar tools